controllers/tasksController.js

```diff
@@ -0,0 +1,47 @@
+const taskQuery = require('../models/tasks')
+/**
+ * Creates new task
+ *
+ * @param req {Object} - Express request object
+ * @param req.body {Object} - Task object
+ * @param res {Object} - Express response object
+ */
+const addNewTask = async (req, res) => {
+  try {
+    const task = await taskQuery.addTask(req.body)
+    return res.json({
+      message: 'Task created successfully!',
+      task: req.body,
+      id: task.taskId
+    })
+  } catch (err) {
+    logger.error(`Error while creating new task: ${err}`)
+    return res.boom.badImplementation('An internal server error occurred')
+  }
+}
+/**
+ * Fetches all the tasks
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const fetchTasks = async (req, res) => {
+  try {
+    const allTasks = await taskQuery.fetchTasks()
+    if (allTasks.length > 0) {
+      return res.status(200).json({
+        message: 'Tasks returned successfully!',
+        tasks: allTasks
+      })
+    }
+    return res.boom.notFound('No tasks found')
```

Comment:
Ideally this should be a 200 response.
Its a success response with no tasks. 🤔

Reply:
So , if there are no tasks .. we still return a success response, but the `tasks` in the response message will be an empty object , is that what you are trying to tell ?

Reply:
Yes, because its actually not a client error, right?

Reply:
Ok Makes sense, Thanks.

Reply:
Still the same. Comment not addressed.

Reply:
It should return an empty tasks array in this case with 200 response.
The client has requested the correct resource in this case.